src/test/java/de/rabea/game/GameSetupTest.java

```diff
@@ -32,6 +33,16 @@ public void setsUpFirstHumanVsHuman3x3Game() {
     }
 
     @Test
+    public void playsHuman3x3GameTwice() {
+        fakeUserInterface.provideConsoleInput("1", "3", "1", "7", "3", "4", "2", "y", "1", "3", "2", "5", "9", "7", "3", "6", "4", "8", "1", "n");
```

Comment:
Hm... good question @christophgockel ! 
I'm not sure yet how the GUI will communicate with the Game but maybe it should also pass numbers, depending on which cell the user clicks? So then the UI double could user integers rather than strings but still numbers.